static/styles/app_components.css

```diff
@@ -563,6 +563,11 @@ div.overlay {
     color: hsl(0, 0%, 100%);
 }
 
+.unread_mention_info:not(:empty) {
```

Comment:
I think a 2px `margin-left` here would fix the overflow issue; test-deploying that but you might want to play with it.

Reply:
Cool, thanks @timabbott , I played with the PR and it looked fine with 2px margin-left;
pushed the changes;